--- pyside-setup-everywhere-src-5.11.2/build_scripts/main.py~	2018-09-07 10:25:52.000000000 +0000
+++ pyside-setup-everywhere-src-5.11.2/build_scripts/main.py	2018-12-04 15:21:51.108731819 +0000
@@ -535,7 +535,7 @@
         py_executable = sys.executable
         py_version = "{}.{}".format(sys.version_info[0], sys.version_info[1])
         py_include_dir = get_config_var("INCLUDEPY")
-        py_libdir = get_config_var("LIBDIR")
+        py_libdir = "/usr/lib64"
         py_prefix = get_config_var("prefix")
         if not py_prefix or not os.path.exists(py_prefix):
             py_prefix = sys.prefix
